src/test/scala/mesosphere/marathon/integration/RemoteRegionOffersIntegrationTest.scala

```diff
@@ -0,0 +1,111 @@
+package mesosphere.marathon
+package integration
+
+import java.util.UUID
+
+import mesosphere.AkkaIntegrationTest
+import mesosphere.marathon.integration.facades.ITEnrichedTask
+import mesosphere.marathon.integration.facades.MarathonFacade.extractDeploymentIds
+import mesosphere.marathon.integration.facades.MesosFacade.ITResourceStringValue
+import mesosphere.marathon.integration.setup._
+import mesosphere.marathon.raml.App
+import mesosphere.marathon.state.{ FaultDomain, PathId, Region, Zone }
+import mesosphere.marathon.state.PathId._
+import mesosphere.mesos.Constraints
+import org.scalatest.Inside
+
+import scala.concurrent.duration._
+
+@IntegrationTest
+class RemoteRegionOffersIntegrationTest extends AkkaIntegrationTest with EmbeddedMarathonTest with Inside {
+
+  override lazy val mesosNumMasters = 1
+  override lazy val mesosNumSlaves = 3
+
+  class Fixture {
+    val homeRegion = Region("home_region")
+    val homeZone = Zone("home_zone")
+
+    val remoteRegion = Region("remote_region")
+    val remoteZone1 = Zone("remote_zone1")
+    val remoteZone2 = Zone("remote_zone2")
+  }
+
+  val fixture = new Fixture
+
+  override def mastersFaultDomains = Seq(Some(FaultDomain(region = fixture.homeRegion, zone = fixture.homeZone)))
+
+  override def agentsFaultDomains = Seq(
+    Some(FaultDomain(region = fixture.remoteRegion, zone = fixture.remoteZone1)),
+    Some(FaultDomain(region = fixture.remoteRegion, zone = fixture.remoteZone2)),
+    Some(FaultDomain(region = fixture.homeRegion, zone = fixture.homeZone)))
+
+  def appId(suffix: String): PathId = testBasePath / s"app-${suffix}"
+
+  "Region Aware marathon" must {
+    "Launch an instance of the app in the default region if region is not specified" in new Fixture {
```

Comment:
[Nit]: I'd avoid this style of fixture creation and use `val f = new Fixture` and `f.homeRegion`. This way it's way easier to read where something is coming from. Imagine someone has now access to IntelliJ and has to be able to understand the code.